pkg/apis/core/types.go

```diff
@@ -4383,8 +4383,22 @@ type ConfigMap struct {
 
 	// Data contains the configuration data.
 	// Each key must consist of alphanumeric characters, '-', '_' or '.'.
+	// The value must be a string (UTF-8). If the value has byte
+	// sequences not in the UTF-8 range, please use the BinaryData field.
+	// The keys stored in Data should not overlap with the keys in
+	// the BinaryData field.
 	// +optional
 	Data map[string]string
+
+	// Data contains the binary data.
+	// Each key must consist of alphanumeric characters, '-', '_' or '.'.
+	// BinaryData can contain byte sequences that are not in the UTF-8 range.
+	// The keys stored in BinaryData should not overlap with the ones in
+	// the Data field. Since BinaryData is a new field (since 1.10), older
```

Comment:
Maybe just say "Requires 1.10+ apiserver and kubelet"

Reply:
Done